bundles/ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java

```diff
@@ -406,28 +406,16 @@ public void doRun(){
 			
 			@Override
 			public void run(){
-				Kontakt kontakt = (Kontakt) ElexisEventDispatcher.getSelected(Kontakt.class);
-				if (kontakt == null) {
-					SWTHelper.showInfo("Kein Kontakt ausgewählt",
-						"Bitte wählen Sie vor dem Drucken einen Kontakt!");
-					return;
+				Command cmd = cmdService.getCommand("ch.elexis.core.ui.commands.printContactLabel");
+				if (!cmd.isDefined()) {
+					  cmd.define(Messages.GlobalActions_PrintContactLabel, Messages.GlobalActions_PrintContactLabelToolTip, cmdCategory);
```

Comment:
why is this necessary? if it is not defined and you would manually define it here - where is the default handler defined?

Reply:
Fixed. The checks are unnecessary because all commands are well-defined in plugin.xml. The default handlers are also defined in plugin.xml.